File: 5KYU/Perimeter of squares in a rectangle/solution.cs
using System;
using System.Numerics;

public class SumFct
{
  public static BigInteger perimeter(BigInteger n) 
  {
    BigInteger total = 0;
    BigInteger a = 4;
    BigInteger b = 4;
    
    for(BigInteger i = 0; i <= n; i++)
    {
      total += a;
      BigInteger c = a + b;
      a = b;
      b = c;
    }
    return total;
  }
}

File: 5KYU/Perimeter of squares in a rectangle/solution.rs
fn perimeter(n: u64) -> u64 {
    let mut total:u64 = 0;
    let mut a:u64 = 4;
    let mut b:u64 = 4;
    
    for x in 0..=n {
      total += a;
      let c:u64 = a+b;
      a = b;
      b = c;
    }
      
    return total;
  }